docs/migration.md

```diff
@@ -47,6 +47,7 @@ consider additional positioning prop support on a case-by-case basis.
 #### @zendeskgarden/react-chrome
 
 - Removed `PRODUCT` type export. Use `IHeaderItemProps['product']` instead.
+- Removed `hasFooter` prop for `Body` (no replacement needed)
```

Comment:
Should we call out that `IBodyProps` is no longer exported?

Reply:
Yep, good call.